tests/data/vdb_basic.py

```diff
@@ -1,5 +1,6 @@
 # import wthell
-from viztracer import VizCounter, VizTracer, VizObject
+from viztracer import VizCounter, VizObject, VizTracer
+
```

Comment:
As we are cleaning code here, could you remove all the comments related to `wthell` or `wth`? I think those are left there when I was debugging.

Reply:
The commented exception raising as well